packages/block-library/src/image/edit.js

```diff
@@ -338,6 +350,10 @@ class ImageEdit extends Component {
 		} ) );
 	}
 
+	setContainer( ref ) {
```

Comment:
I'd encourage using `createRef` for this instead, largely because it requires less code to implement.

https://reactjs.org/docs/refs-and-the-dom.html

Reply:
I can make this change, initially it did not work for me but I realize that was due to the fact that I was trying to use the `<Fragment>` which does not work at all.